2015/RDO-Meetups.yml

```diff
@@ -4,176 +4,6 @@ tags: RDO
 
 talks:
 
```

Comment:
Why did all the previous talks go away?

The calendar handles this all for us, and it's good to have previous things archived. Things should only be deleted if they didn't happen at all.

I should also now have sorting of talks too, so you could even add them in reverse order if that makes things easier to see. (If this doesn't work for whatever reason, as sorting is new, then notify me / file a bug.) Or just keep adding at the end of the file.

What will happen:

1. Sites will rebuild with the new info (when the sites are updated to point to the new update of the submodule, of course)
2. Previous events will fall away from the events page (they will be skipped when rebuilding the page)
3. Previous events will still display in the calendar widget as old events, and they will link to the yearly archived version
4. The .ics file will still list the past events (as usual), so events won't magically disappear from people's subscribed calendars

(If anything doesn't happen like the above, then it is a bug, so let me know and I'll fix it.)